examples/emr/v1beta1/cluster.yaml

```diff
@@ -0,0 +1,504 @@
+apiVersion: emr.aws.upbound.io/v1beta1
```

Comment:
I'm asking to be sure if all the below-added resources are necessary for the cluster?

Reply:
I checked all of them again and removed some.
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/emr_cluster#bootable-cluster
These are the resources for the bootable cluster from Terraform.